ons-spark/raw-notebooks/reading-and-writing-data/reading-and-writing-data-spark.ipynb

```diff
@@ -422,15 +424,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Note that unlike other methods, Spark doesn't have a built in `spark.read.avro()` method so we need to use a slightly different method to read this in, by first specifying the format as \"avro\" and then using `.load()` to read in the file. Note that this method would also work for other formats, such as `spark.read.format(\"parquet\").load(...)` but is slightly more verbose than the other methods demonstrated. \n",
+    "```r\n",
+    "library(sparkavro)\n",
+    "\n",
+    "animal_rescue = sparkavro::spark_read_avro(sc, \"animal_rescue\", config$rescue_path_avro)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
     "\n",
     "### Writing out Avro files\n",
-    "To write out an Avro file, you can use [`dataframe.write.format(\"avro\").save()`](https://sparkbyexamples.com/spark/read-write-avro-file-spark-dataframe/). Note that like the method to read in Avro files, we need to specify the format and then use `save` to specify that we want to save this out."
+    "To write out an Avro file, you can use [`dataframe.write.format(\"avro\").save()`](https://sparkbyexamples.com/spark/read-write-avro-file-spark-dataframe/). Note that like the method to read in Avro files, we need to specify the format and then use `save` to specify that we want to save this out.\n",
```

Comment:
I think there may be a typo at the end of the sentence, is it supposed to say 'that we want to save this output' instead of just 'out'.